pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md

```diff
@@ -372,6 +372,8 @@ The Dex Addon acts as the cluster's OpenID Connect identity provider. You must c
 
```

Comment:
```suggestion
## Set up Proxy Overrides

When adopting the DKP cluster, there is currently not a parameter to include the proxy override secret. Therefore, the preprovisioned machine templates must be manually updated to include it.
Update Control Plane Preprovisioned Machine Template
`kubectl edit preprovisionedmachinetemplate kaas-1348-cve-int-control-plane --kubeconfig=admin.conf`

1.  Create the HTTP proxy override file

1.  Create the overrides secret & label it from the HTTP proxy override file

1.  Edit the preprovisioned machine templates for the control plane & worker nodes with the overrides secret created in the previous step (as explained in the above section)
```

Reply:
pre-provisioned

Reply:
Gah, hyphenated here, not hyphenated in the current docs, I can't keep up! :P

Reply:
@dkoshkin @cneth @msdolbey 
Anthony gave me these steps & code above - please verify! Thanks

Reply:
these steps looks good to me here

Reply:
👀

Reply:
I updated the code blocks to make it a bit more 'fancy' when rendering in the browser. 👍

Reply:
will the ${CLUSTER_NAME} env variable actual hold the set value when directly editing the cm.yaml file tho?

Reply:
I think its more of a place holder than anything. I don't think it will hold

Reply:
@bovan01 - that variable will need to be manually replaced when completing the process.

Reply:
yep, that was my concern with referencing as an env variable within the step to edit the yaml file, could be misconstrued unless specifically pointed out that it needs to be manually updated with that value